src/engine/engine_wb.c

```diff
@@ -42,6 +43,9 @@ static void _ocf_write_wb_update_bits(struct ocf_request *req)
 
 		ocf_req_hash_unlock_wr(req);
 	}
+
+	if (ocf_engine_mapped_count(req) > 0)
```

Comment:
So you will not call "set_hot" in case of full miss? I think it's incorrect.

Reply:
Ok, it actually might make sense to keep track of write operations for the purpose of cleaning

Reply:
Ok, I've changed the code to call set_hot() unconditionally because I couldn't came up with any scenario when a write IO would be submitted in WB and the timestamp shouldn't be updated.